# vector of Vector3 pointing to options for driving
# Vector3 -> radius, theta (relative to robot), observed width of space
geometry_msgs/Vector3[] spaces
